src/dbnode/persist/fs/commitlog/iterator.go

```diff
@@ -72,14 +74,17 @@ func NewIterator(iterOpts IteratorOpts) (iter Iterator, corruptFiles []ErrorWith
 	filteredCorruptFiles := filterCorruptFiles(corruptFiles, iterOpts.FileFilterPredicate)
 
 	scope := iops.MetricsScope()
+	log := iops.Logger()
```

Comment:
Nit: might want to move `log` (and `scope`) into a single `var` block at the top of this function?